common/conf.py

```diff
@@ -170,7 +170,7 @@ def get_previous_version(majornumber, minornumber=0):
     'zerocdoc' : ('https://doc.zeroc.com/%s', ''),
     'djangodoc' : ('https://docs.djangoproject.com/en/1.8/%s', ''),
     'doi' : ('http://dx.doi.org/%s', ''),
-    'pypi': ('https://pypi.python.org/pypi/%s', ''),
+    'pypi': ('https://pypi.org/%s', ''),
```

Comment:
Should it be `https://pypi.org/project/%s` e.g. https://pypi.org/project/omero-marshal/?